src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTypeTableViewCell.swift

```diff
@@ -42,16 +42,23 @@ class DynamicTypeTableViewCell: UITableViewCell, DynamicTableViewTextCell {
 	func configureDynamicType(size: CGFloat = 17, weight: UIFont.Weight = .regular, style: UIFont.TextStyle = .body) {
 		contentTextLabel.font = UIFont.preferredFont(forTextStyle: style).scaledFont(size: size, weight: weight)
 		contentTextLabel.adjustsFontForContentSizeCategory = true
-		contentTextLabel.numberOfLines = 0
+		contentTextLabel.lineBreakMode = .byTruncatingTail
+		contentTextLabel.numberOfLines = 4
 	}
 
 	func configure(text: String, color: UIColor? = nil) {
 		contentTextLabel.text = text
 		contentTextLabel.textColor = color ?? .enaColor(for: .textPrimary1)
+		
+		let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(labelTapped))
+		contentTextLabel.addGestureRecognizer(tapGestureRecognizer)
+		contentTextLabel.isUserInteractionEnabled = true
+		contentTextLabel.isAccessibilityElement = true
```

Comment:
That `internal func` is declared to configure only a text and a color. 

To configure a special gesture + handler, you should better provide a separate configure function for that. 

Of if the gesture behaviour shall be generic / common valid, then use another private place in the class to configure it. :)